.travis.yml

```diff
@@ -18,7 +18,6 @@ sudo: false
 
 jdk:
   - oraclejdk8
-  - openjdk7
```

Comment:
Do you want to add OpenJDK 8 here as well, or is that not a supported / recommended JDK distribution?

Reply:
Sure, I'll add that in. Cloudera still says only to use the Oracle JDK for things, but in practice, OpenJDK is gaining popularity (and converging) anyway.

Thanks!